### ISSUE TITLE
flatpickr does not display month

### ISSUE THREAD

**Issue by wlnirvana** [autolab/Autolab#1153]
Visit [the demo site][1] and click "Due at". There is only "2020" and no month for selection.

![image](https://user-images.githubusercontent.com/3822785/81946619-2ca89b80-9632-11ea-806d-755a5e3f0d80.png)

[1]:http://autolab.ml/courses/AutoPopulated/assessments/labtemplate/edit

**Comment by wlnirvana** [comment]
It seems flatpickr [implements][2] the month dropdown menu as something like 

```html
<select class="flatpickr-monthDropdown-months" tabindex="-1">
...
</select>
```

But `materialize-sass` happens to [set `display: none` on HTML `select` elements][1], thus hiding the dropdown menu.

[1]:https://github.com/mkhairi/materialize-sass/blob/a825c731db483fcba61ddeadc0230bd82b5a6c53/assets/stylesheets/materialize/components/forms/_select.scss#L4
[2]:https://flatpickr.js.org/examples/#basic

**Comment by victorhuangwq** [comment]
@wlnirvana thank you for reporting this! I have checked out the master branch and verified that the problem exist. I have addressed the issue in #1154 , and after it is reviewed it will be pushed in. 

It won't be reflected in the demo-site immediately however, as we are pausing rollouts into the demo-site for this month.
